src/core.c/Rat.pm6

```diff
@@ -83,6 +93,7 @@ multi sub CREATE_RATIONAL_FROM_INTS(Int:D $nu, Int:D $de, Any, Any) is raw {
            nqp::p6bindattrinvres(nqp::create(Rat),Rat,'$!numerator',$nu),
            Rat,'$!denominator',$de
          )
+      !! $*RAT-UPGRADE-POLICY.UPGRADE-RAT(nu, de)
       !! nqp::p6box_n(nqp::div_In($nu,$de))  # downgrade to float
```

Comment:
Syntax error here, looks like you meant to replace the `nqp::p6box_n(nqp::div_In($nu,$de))  # downgrade to float` with `$*RAT-UPGRADE-POLICY.UPGRADE-RAT(nu, de)`, but accidentally just copied it in.